composer.json

```diff
@@ -121,6 +123,9 @@
         }
     ],
     "scripts": {
+        "post-install-cmd": [
+            "bin/console assets:install --symlink"
+        ],
```

Comment:
https://symfony.com/doc/master/bundles/NelmioApiDocBundle/faq.html#asset-files-not-loaded